addons/binding/org.openhab.binding.pixometer/src/main/java/org/openhab/binding/pixometer/internal/config/Annotation.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pixometer.internal.config;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link Annotation} class is the representing java model for the json result for a annotation from the pixometer
+ * api
+ *
+ * @author Jerome Luckenbach - Initial Contribution
+ *
+ */
+public class Annotation {
+
+    @SerializedName("id")
+    @Expose
+    private Integer id;
+    @SerializedName("rectangle")
+    @Expose
+    private Object rectangle;
+    @SerializedName("meaning")
+    @Expose
+    private String meaning;
+    @SerializedName("text")
+    @Expose
+    private String text;
+    @SerializedName("image")
+    @Expose
+    private Integer image;
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public Object getRectangle() {
+        return rectangle;
+    }
+
+    public void setRectangle(Object rectangle) {
+        this.rectangle = rectangle;
+    }
+
+    public String getMeaning() {
+        return meaning;
+    }
+
+    public void setMeaning(String meaning) {
+        this.meaning = meaning;
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+
+    public Integer getImage() {
+        return image;
+    }
+
+    public void setImage(Integer image) {
+        this.image = image;
+    }
+
+}
```

Comment:
Missing new line at end of the file (This applies to more files, GitHub has a nice indicator to show to which files it applies)